### PR TITLE
TM-4196: moving clear filters to above inactive toggle

### PR DESCRIPTION
**Pull request by pveland621**
ACs:
1. Verify clear filters button is above inactive toggle and still works

### CODE REVIEW

File: src/Components/Agenda/EmployeeAgendaSearch/EmployeeAgendaSearch.jsx [hunk 1]
```diff
@@ -315,6 +315,16 @@ const EmployeeAgendaSearch = ({ isCDO, viewType }) => {
                 <div className="search-header">
                   Search For An Employee
                 </div>
+                <div className="filterby-container">
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearch/EmployeeAgendaSearch.jsx:318]: 🥳

File: src/sass/_employeeSearch.scss [hunk 2]
```diff
@@ -472,5 +472,5 @@
 }
 
 .eas-inactive-toggle {
-  margin-top: 25px;
+  margin-top: 15px;
```
Comment (SidelineCory24) [inline src/sass/_employeeSearch.scss:475]: lets do 20px to give it a little more breathing room

Comment (pveland621) [reply src/sass/_employeeSearch.scss:475]: done!

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go!🚀

File: src/sass/_employeeSearch.scss [hunk 3]
```diff
@@ -60,6 +60,7 @@
       align-items: center;
       display: flex;
       margin-right: 10px;
+      padding-left: 5px;
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:63]: might make more sense to add this to `empl-search-filters`

**Review by elizabeth-jimenez — APPROVED** [review]

File: src/sass/_employeeSearch.scss (hunk 3 above)
Comment (scott062) [reply src/sass/_employeeSearch.scss]: Good catch - done

**Pull request merged**